fluentd/run.sh

```diff
@@ -279,6 +279,15 @@ if type -p jemalloc-config > /dev/null 2>&1 && [ "${USE_JEMALLOC:-}" = true ] ;
     export LD_PRELOAD=$( jemalloc-config --libdir )/libjemalloc.so.$( jemalloc-config --revision )
 fi
 
+if [ "${USE_JOURNAL:-}" = "true" ] ; then
+  # Include DEBUG log level messages when collecting from journald
+  # https://bugzilla.redhat.com/show_bug.cgi?id=1505602
+  if [ "${COLLECT_JOURNAL_DEBUG_LOGS:-false}" = true ] ; then
```

Comment:
Shouldn't this default to `true`?  Don't we want to collect everything by default, so that if an admin does not want debug messages collected, they can set `COLLECT_JOURNAL_DEBUG_LOGS` to `false`?

Reply:
Changed to `true`.